test/functional/test_runner.py

```diff
@@ -477,6 +477,11 @@ def get_next(self):
                               log_stderr))
         if not self.jobs:
             raise IndexError('pop from empty list')
+
+        # Print remaining jobs when the backlog is empty.
```

Comment:
Could be more clear - first usage of "backlog".

Reply:
better?